source/helpers/hotfix.ts

```diff
@@ -16,21 +18,30 @@ function parseCsv(content: string): string[][] {
 	return lines;
 }
 
-export type HotfixStorage = Array<[FeatureID, string]>;
-
-export const updateHotfixes = cache.function(async (version: string): Promise<HotfixStorage> => {
+async function fetchHotfix(path: string): Promise<string> {
 	// The explicit endpoint is necessary because it shouldn't change on GHE
 	// We can't use `https://raw.githubusercontent.com` because of permission issues https://github.com/refined-github/refined-github/pull/3530#issuecomment-691595925
-	const request = await fetch('https://api.github.com/repos/refined-github/refined-github/contents/hotfix.csv?ref=hotfix');
+	const request = await fetch(`https://api.github.com/repos/refined-github/yolo/contents/${path}`);
 	const {content} = await request.json();
 
 	// Rate-limit check
+	if (content) {
+		return atob(content).trim();
+	}
+
+	return '';
+}
+
+export type HotfixStorage = Array<[FeatureID, string]>;
```

Comment:
I think eventually we should have a single `updater` function and a single `hotfixes` object in the options, like:

```js
{
	style: 'body {color: red}',
  broken: ['forked-to'],
  strings: {from: 'to'},
}
```